index.html

```diff
@@ -1239,37 +1257,57 @@ <h2>
             `"hidden"`, return [=a promise rejected with=] an
             "{{InvalidStateError}}" {{DOMException}}.
             </li>
-            <li>Let |resetResultPromise:Promise| be [=a new promise=].
+            <li>Increment [=this=].{{GamepadHapticActuator/[[sequenceId]]}}.
             </li>
             <li>If [=this=].{{GamepadHapticActuator/[[playingEffectPromise]]}}
-            is not `null`, do the following steps [=in parallel=]:
+            is not `null`:
               <ol>
                 <li>Let |effectPromise| be
                 [=this=].{{GamepadHapticActuator/[[playingEffectPromise]]}}.
                 </li>
+                <li>Set
+                [=this=].{{GamepadHapticActuator/[[playingEffectPromise]]}} to
+                `null`.
+                </li>
+                <li>[=Queue a global task=] on the [=relevant global object=]
+                of [=this=] using the [=gamepad task source=] to [=resolve=]
+                |effectPromise| with {{GamepadHapticsResult/"preempted"}}.
+                </li>
+              </ol>
+            </li>
+            <li>Let [=this=].{{GamepadHapticActuator/[[playingEffectPromise]]}}
```

Comment:
consider adding a step asserting nullness of the promise at this point(for clarity).

Reply:
Done